### ISSUE TITLE
Update word lists regularly

### ISSUE THREAD

**Issue by theseion** [coreruleset/coreruleset#2621]
1. get an idea from where to take input, add it as comment to the file
2. manually update the file first
3. automate 2)
4. define a cadence for updating

The end goal is to automate as much as possible, so that our word lists don't go stale.

Decision from https://github.com/coreruleset/coreruleset/issues/2539.

### Java-related updates

- [x] #2639
- [ ] #2640 [POSTPONED AFTER V4]
- [ ] #2653 [POSTPONED AFTER V4]

### Scanners/Crawlers

- [X] #2638  (PR exists: #3236)
- [X] #2645  (PR exists: #3236) 
- [X] #2646  (PR exists: #3236)
- [X] #2647 (PR exists: #3241)
- [X] #2648  (PR exists: #3235)

### RCE Rules

- [x] #2643
- [x] #2644
- [x] #2641
- [x] #2628
- [x] #2630
- [x] #2673
- [x] #2672
- [x] #2692
- [x] #2694

### Windows

- [x] #2629
- [x] #2682
- [x] #2671
- [x] #2693

### PHP

- [x] #2650
- [x] #2652
- [x] #2651

#### PHP functions rules

These next four are dependent on each other. We need to review all PHP functions and redo the classification in the 4 groups. Documenting the process, the decisions on why some words are in PLX, and maybe converge to something that can be done in an automated way.

- [x] #2684
- [x] #2685
- [x] #3021
- [x] #3022

### SQL Errors

- [ ] #2642 [POSTPONED AFTER V4]

### Others

- [x] #2631
- [x] #2649
- [x] #2698
- [x] #2724
- [x] #2800 

In summary, word list consist of all `.data` files in the `rules/` directory, and the 932XXX files mentioned here.

**Comment by dune73** [comment]
I have added the remark "[POSTPONED AFTER V4]" to #2653 since it deals with java response rules.

**Comment by fzipi** [comment]
Closing this one as completed. I'll create a followup ticket for documenting the next steps.
